_posts/2020-07-22-internal-projects.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Internal Projects"
+date: 2020-07-28 12:00:00
+categories: ["ombulabs learning"]
+author: rdormer
+---
+
+With the constant pressure to both find and then execute on client
+projects, agencies often lose sight of the possibility of working on
+internal projects. While the initial reaction is often to dismiss them
+as a distraction from the important client facing work, carefully
+chosen interal projects can be very valuable. Aside from the obvious
+benefit of solving internal problems that might not have a readily
+available solution, they also provide training opportunities for
+developers. They give you a chance to try new ways of
+doing things with relatively low risk, or perhaps preview new
+languages and frameworks you've been considering.
+
+<!--more-->
+
+Here at OmbuLabs, we use internal projects for all of these reasons,
+and we've found them to be a valuable part of our company culture.
+Here's a brief sampling of the most prominent ones we've been working
+on:
+
+Dash - While Github and Pivotal integrate reasonably well, we found
+that what they provided didn't quite match our desired workflow. We
+needed more than just a way to link Github activity to Pivotal
+stories.  We wanted to have an overview of both in dashboard form.
+Dash lets us see our open stories and PRs together in one place.
+
+Points - As a development agency, estimating projects is a critically
+important task, and one that's worth spending significant effort to
+get right. Getting estimates wrong has consequences in any business,
+but when you're doing the work for other people, those consequences
+can be especially bad. One thing we quickly figured out is that when
+it comes to estimates, two heads are better than one. Points is a tool
+that we developed so that multiple developers, typically the ones who
+will be doing the actual work, can weigh in with their estimates. The
+band of estimated values that develops has turned out to be
+surprisingly accurate in most cases - and those cases where it hasn't
+have invariably been important learning opportunities.
```

Comment:
Are we planning to open source this project? If so, we can mention that since there's no link to look at it

Reply:
@lubc I don't believe we're planning to open source it right now